### PR TITLE
pass options from module config to wedia content picker

### PR DESCRIPTION
**Pull request by ValJed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,15 +28,18 @@ To configure the modules in this bundle, you should add `apostrophe-media-source
           src: 'https://xyz.wedia-group.com/asset-picker/wedia-content-picker.js',
           server: 'https://xyz.wedia-group.com',
           name: 'WediaContentPicker',
-          maxConcurrentImports: 20 // optional - activates the multiple import -
+          options: {} // optional - pass options to the wedia content picker -}
```
Comment (falkodev) [inline README.md:31]: Why a final `}` on this line?

```
wedia content picker -}
```

File: README.md [hunk 2]
```diff
@@ -28,15 +28,18 @@ To configure the modules in this bundle, you should add `apostrophe-media-source
...
       }
     }
   }
 ```
 
-Note: to access the `Unsplash` API you will need to create a free developer account, which will provide you with an access key. For Wedia you will need to work with the Wedia team.
+Notes:
+* You can find existing wedia options in the documentation: https://crossmedia.atlassian.net/wiki/spaces/WD/pages/379551745/Wedia+Content+Picker#Settings
+
+* to access the `Unsplash` API you will need to create a free developer account, which will provide you with an access key, this free account is limited to 50 requests per hour. For Wedia you will need to work with the Wedia team.
```
Comment (falkodev) [inline README.md:42]: If you start the previous paragraph with an uppercase letter, I think this one needs it too.

**Review by falkodev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ValJed) [reply README.md]: Removed

File: README.md (hunk 2 above)
Comment (ValJed) [reply README.md]: Yup, fixed

**Review by falkodev — APPROVED** [review]

**Pull request merged**